### ISSUE TITLE
A11y_Fluent UI_Components_Alert_ScreenReader: Narrator is not announcing close button it is announcing as button.

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3397]
**GitHubTags:**

#A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG4.1.2.
#Screenreader; #A11yMAS;#Narrator.

**OS Details:**

Web: win11

**Environment Details:**

Application Name: Fluent Ui

**Repro Steps:**

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Components" button and hit enter.
4.Tab till "Alert" link on the left side bar and hit enter.
5.verify Narrator is not announcing close button it is announcing as button.

**Actual Result**

The narrator is not announcing the close button it is announcing as button.

**Expected Result:**

The narrator should announce as close button instead of announcing as button.

**User Impact:**

Screen reader users will face difficulties when narrator is not announcing the close button it is announcing as button.

![3397A11y_Fluent UI_Components_Alert_ScreenReader](https://user-images.githubusercontent.com/89508906/198257257-f5bc3d44-24c0-47f7-9163-dd1ced7d34b5.gif)

**Comment by Latha2564** [comment]
Closing this bug as bug logged in wrong ADO
